kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1049,4 +1049,20 @@ object KyuubiConf {
       .transform(_.toUpperCase(Locale.ROOT))
       .checkValues(OperationModes.values.map(_.toString))
       .createWithDefault(OperationModes.NONE.toString)
+
+  object OperationLanguages extends Enumeration {
+    type OperationLanguage = Value
+    val SQL, SCALA = Value
+  }
+
+  val OPERATION_LANGUAGE: ConfigEntry[String] =
```

Comment:
We have a ENGINE_TPYE which can be SPARK_SQL and FLINK_SQL, seems not accurate now

Reply:
it seems that for OperationLanguage option, the operations can share the same engine.
It looks better.